--- Trac-1.0beta1/tracopt/mimeview/silvercity.py~	2012-08-06 23:44:52.000000000 +0300
+++ Trac-1.0beta1/tracopt/mimeview/silvercity.py	2012-08-12 21:56:43.786696547 +0300
@@ -48,6 +48,7 @@
     'application/x-yaml':       ('YAML', 3),
     'text/x-yaml':              ('YAML', 3),
     'application/x-javascript': ('CPP', 3), # Kludgy.
+    'application/javascript':   ('CPP', 3), # Kludgy.
     'text/x-asp':               ('HyperText', 3, {'asp.default.language':2}),
     'text/x-c++hdr':            ('CPP', 3),
     'text/x-c++src':            ('CPP', 3),
@@ -60,6 +61,7 @@
     'application/x-httpd-php3': ('HyperText', 3, {'asp.default.language': 4}),
     'text/x-java':              ('Java', 3),
     'text/x-javascript':        ('CPP', 3), # Kludgy.
+    'text/javascript':          ('CPP', 3), # Kludgy.
     'text/x-psp':               ('HyperText', 3, {'asp.default.language': 3}),
     'text/x-python':            ('Python', 3),
     'text/x-ruby':              ('Ruby', 3),
--- Trac-1.0beta1/tracopt/mimeview/enscript.py~	2012-08-06 23:44:52.000000000 +0300
+++ Trac-1.0beta1/tracopt/mimeview/enscript.py	2012-08-12 21:57:48.773049013 +0300
@@ -30,6 +30,7 @@
     'application/postscript':   ('postscript', 2),
     'application/x-csh':        ('csh', 2),
     'application/x-javascript': ('javascript', 2),
+    'application/javascript':   ('javascript', 2),
     'application/x-troff':      ('nroff', 2),
     'text/html':                ('html', 2),
     'text/x-ada':               ('ada', 2),
@@ -49,6 +50,7 @@
     'text/x-inf':               ('inf', 2),
     'text/x-java':              ('java', 2),
     'text/x-javascript':        ('javascript', 2),
+    'text/javascript':          ('javascript', 2),
     'text/x-ksh':               ('ksh', 2),
     'text/x-lua':               ('lua', 2),
     'text/x-m4':                ('m4', 2),
