diff -Naur trac-0.8.1.orig/scripts/trac-admin trac-0.8.1/scripts/trac-admin
--- trac-0.8.1.orig/scripts/trac-admin	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/scripts/trac-admin	2005-05-29 23:44:04.000000000 +0200
@@ -484,7 +484,7 @@
             project_name = arg[0]
             repository_dir = arg[1]
             templates_dir = arg[2]
-        from svn import util, repos, core
+        from svn import repos, core
         core.apr_initialize()
         pool = core.svn_pool_create(None)
         try:
diff -Naur trac-0.8.1.orig/trac/Browser.py trac-0.8.1/trac/Browser.py
--- trac-0.8.1.orig/trac/Browser.py	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/trac/Browser.py	2005-05-29 23:45:22.000000000 +0200
@@ -74,20 +74,20 @@
 
             created_rev = svn.fs.node_created_rev(root, fullpath, self.pool)
             date = svn.fs.revision_prop(self.fs_ptr, created_rev,
-                                    svn.util.SVN_PROP_REVISION_DATE,
+                                    svn.core.SVN_PROP_REVISION_DATE,
                                     self.pool)
             if date:
-                date_seconds = svn.util.svn_time_from_cstring(date,
+                date_seconds = svn.core.svn_time_from_cstring(date,
                                                           self.pool) / 1000000
                 date = time.strftime('%x %X', time.localtime(date_seconds))
             else:
                 date_seconds = 0
                 date = ''
             author = svn.fs.revision_prop(self.fs_ptr, created_rev,
-                                      svn.util.SVN_PROP_REVISION_AUTHOR,
+                                      svn.core.SVN_PROP_REVISION_AUTHOR,
                                       self.pool)
             change = svn.fs.revision_prop(self.fs_ptr, created_rev,
-                                             svn.util.SVN_PROP_REVISION_LOG,
+                                             svn.core.SVN_PROP_REVISION_LOG,
                                              self.pool)
             item = {
                 'name'         : name,
diff -Naur trac-0.8.1.orig/trac/Changeset.py trac-0.8.1/trac/Changeset.py
--- trac-0.8.1.orig/trac/Changeset.py	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/trac/Changeset.py	2005-05-29 23:45:55.000000000 +0200
@@ -60,7 +60,7 @@
         # and the new version uses the same charset, not always the case
         # but that's all we can do...
         mime_type = svn.fs.node_prop (self.new_root, new_path,
-                                      svn.util.SVN_PROP_MIME_TYPE,
+                                      svn.core.SVN_PROP_MIME_TYPE,
                                       pool)
         # We don't have to guess if the charset is specified in the
         # svn:mime-type property
diff -Naur trac-0.8.1.orig/trac/core.py trac-0.8.1/trac/core.py
--- trac-0.8.1.orig/trac/core.py	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/trac/core.py	2005-05-29 23:49:36.000000000 +0200
@@ -447,7 +447,7 @@
             svn.core.svn_pool_destroy(pool)
 
 def open_svn_repos(repos_dir):
-    from svn import util, repos, core
+    from svn import repos, core
 
     core.apr_initialize()
     pool = core.svn_pool_create(None)
diff -Naur trac-0.8.1.orig/trac/File.py trac-0.8.1/trac/File.py
--- trac-0.8.1.orig/trac/File.py	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/trac/File.py	2005-05-29 23:48:38.000000000 +0200
@@ -307,7 +307,7 @@
 
         # Try to do an educated guess about the mime-type
         self.mime_type = svn.fs.node_prop (root, self.path,
-                                           svn.util.SVN_PROP_MIME_TYPE,
+                                           svn.core.SVN_PROP_MIME_TYPE,
                                            self.pool)
         if not self.mime_type:
             self.mime_type = self.env.mimeview.get_mimetype(filename=self.path) or \
@@ -323,9 +323,9 @@
 
         self.length = svn.fs.file_length(root, self.path, self.pool)
         date = svn.fs.revision_prop(self.fs_ptr, self.rev,
-                                svn.util.SVN_PROP_REVISION_DATE, self.pool)
-        date_seconds = svn.util.svn_time_from_cstring(date, self.pool) / 1000000
+                                svn.core.SVN_PROP_REVISION_DATE, self.pool)
+        date_seconds = svn.core.svn_time_from_cstring(date, self.pool) / 1000000
         self.last_modified = time.strftime("%a, %d %b %Y %H:%M:%S GMT",
                                       time.gmtime(date_seconds))
         fd = svn.fs.file_contents(root, self.path, self.pool)
-        self.read_func = lambda x, f=fd: svn.util.svn_stream_read(f, x)
+        self.read_func = lambda x, f=fd: svn.core.svn_stream_read(f, x)
diff -Naur trac-0.8.1.orig/trac/Log.py trac-0.8.1/trac/Log.py
--- trac-0.8.1.orig/trac/Log.py	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/trac/Log.py	2005-05-29 23:49:10.000000000 +0200
@@ -43,7 +43,7 @@
                 self.branch_info.setdefault(rev, []).append((change.copyfrom_path, newpath))
 
         shortlog = util.shorten_line(util.wiki_escape_newline(log))
-        t = svn.util.svn_time_from_cstring(date, pool) / 1000000
+        t = svn.core.svn_time_from_cstring(date, pool) / 1000000
         gmt = time.gmtime(t)
         item = {
             'rev'      : rev,
diff -Naur trac-0.8.1.orig/trac/sync.py trac-0.8.1/trac/sync.py
--- trac-0.8.1.orig/trac/sync.py	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/trac/sync.py	2005-05-29 23:51:31.000000000 +0200
@@ -19,7 +19,7 @@
 #
 # Author: Jonas Borgström <jonas@edgewall.com>
 
-from svn import fs, util, delta, repos, core
+from svn import core, fs, delta, repos
 
 def sync(db, repos, fs_ptr, pool):
     """
@@ -27,10 +27,10 @@
     the repository.
     """
 
-    if util.SVN_VER_MAJOR < 1:
+    if core.SVN_VER_MAJOR < 1:
         raise EnvironmentError, \
               "Subversion >= 1.0 required: Found %d.%d.%d" % \
-              (util.SVN_VER_MAJOR, util.SVN_VER_MINOR, util.SVN_VER_MICRO)
+              (core.SVN_VER_MAJOR, core.SVN_VER_MINOR, core.SVN_VER_MICRO)
 
     cursor = db.cursor()
     cursor.execute('SELECT ifnull(max(rev), 0) FROM revision')
@@ -42,13 +42,13 @@
     subpool = core.svn_pool_create(pool)
     for rev in range(num):
         message = fs.revision_prop(fs_ptr, rev + offset,
-                                   util.SVN_PROP_REVISION_LOG, subpool)
+                                   core.SVN_PROP_REVISION_LOG, subpool)
         author = fs.revision_prop(fs_ptr, rev + offset,
-                                  util.SVN_PROP_REVISION_AUTHOR, subpool)
+                                  core.SVN_PROP_REVISION_AUTHOR, subpool)
         date = fs.revision_prop(fs_ptr, rev + offset,
-                                util.SVN_PROP_REVISION_DATE, subpool)
+                                core.SVN_PROP_REVISION_DATE, subpool)
 
-        date = util.svn_time_from_cstring(date, subpool) / 1000000
+        date = core.svn_time_from_cstring(date, subpool) / 1000000
         
         cursor.execute ('INSERT INTO revision (rev, time, author, message) '
                         'VALUES (%s, %s, %s, %s)', rev + offset, date,
diff -Naur trac-0.8.1.orig/trac/util.py trac-0.8.1/trac/util.py
--- trac-0.8.1.orig/trac/util.py	2005-05-29 23:43:18.000000000 +0200
+++ trac-0.8.1/trac/util.py	2005-05-29 23:51:56.000000000 +0200
@@ -32,8 +32,8 @@
 CRLF = '\r\n'
 
 def svn_date_to_string(date, pool):
-    from svn import util
-    date_seconds = util.svn_time_from_cstring(date,
+    from svn import core
+    date_seconds = core.svn_time_from_cstring(date,
                                               pool) / 1000000
     return time.strftime('%x %X', time.localtime(date_seconds))
 
