--- Trac-1.0.1/trac/web/chrome.py~	2013-02-04 13:20:00.000000000 +0200
+++ Trac-1.0.1/trac/web/chrome.py	2013-02-04 13:20:33.277174060 +0200
@@ -303,7 +303,7 @@
     logo_link = Option('header_logo', 'link', '',
         """URL to link to, from the header logo.""")
 
-    logo_src = Option('header_logo', 'src', 'site/your_project_logo.png',
+    logo_src = Option('header_logo', 'src', 'common/trac_banner.png',
         """URL of the image to use as header logo.
         It can be absolute, server relative or relative.
 
@@ -448,7 +448,7 @@
         Only specifying `your-logo.png` is equivalent to the latter.""")
 
     logo_alt = Option('header_logo', 'alt',
-        "(please configure the [header_logo] section in trac.ini)",
+        "",
         """Alternative text for the header logo.""")
 
     logo_width = IntOption('header_logo', 'width', -1,
